0,1
cd1268a5,job_<*>_0483Job Transitioned from <*> to <*>
3592ea3d,Instantiated MRClientService at cf6e5ca2712c<*>
468d66a5,Logging initialized @2155ms to org.eclipse.jetty.util.log.Slf4jLog
16a015cc,jetty-<*>.<*>.<*>.v20230217; built: <*>-<*>-17T08:<*>:<*>.309Z; git: b45c405e4544384de066f814ed42ae3dceacdd49; jvm <*>.<*>.<*>+<*>-post-Ubuntu-1ubuntu120.<*>
0fcdc0c2,node0 Scavenging every 600000ms
ca519185,"Started ServerConnector@7b5b5bfe{HTTP/<*>.<*>, (http/<*>.<*>)}{<*>}"
f9c99537,Assigned container container_<*>_<*>_<*>_<*> to attempt_<*>_<*>_m_<*>_<*>
acd0086c,Processing the event EventType: <*> for container container_<*>_<*>_<*>_<*> taskAttempt attempt_<*>_<*>_m_<*>_<*>
5f64a848,Launching attempt_<*>_<*>_m_<*>_<*>
b088af87,Shuffle port returned by ContainerManager for attempt_<*>_<*>_m_<*>_<*> : <*>
f9ac23df,TaskAttempt: [attempt_<*>_<*>_m_<*>_<*>] using containerId: [container_<*>_<*>_<*>_<*> on NM: <*>
80625245,ATTEMPT_START task_<*>_<*>_m_<*>
3e0cc33b,JVM with ID : jvm_<*>_<*>_m_<*> asked for a task
17ddc538,JVM with ID: jvm_<*>_<*>_m_<*> given task: attempt_<*>_<*>_m_<*>_<*>
d5bff8b5,Progress of TaskAttempt attempt_<*>_<*>_m_<*>_<*> is : <*>.<*>
8febb748,"Task: attempt_<*>_<*>_m_<*>_<*> - exited : java.lang.NumberFormatException: For input string: ""LoGvv"" at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:<*>) at java.base/java.lang.Integer.parseInt(Integer.java:<*>) at java.base/java.lang.Integer.parseInt(Integer.java:<*>) at org.apache.hadoop.conf.Configuration.getInt(Configuration.java:<*>) at org.apache.hadoop.fs.ChecksumFileSystem.setConf(ChecksumFileSystem.java:<*>) at org.apache.hadoop.util.ReflectionUtils.setConf(ReflectionUtils.java:<*>) at org.apache.hadoop.util.ReflectionUtils.newInstance(ReflectionUtils.java:<*>) at org.apache.hadoop.fs.FileSystem.createFileSystem(FileSystem.java:<*>) at org.apache.hadoop.fs.FileSystem.access$<*>(FileSystem.java:<*>) at org.apache.hadoop.fs.FileSystem$Cache.getInternal(FileSystem.java:<*>) at org.apache.hadoop.fs.FileSystem$Cache.get(FileSystem.java:<*>) at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:<*>) at org.apache.hadoop.fs.FileSystem.getLocal(FileSystem.java:<*>) at org.apache.hadoop.fs.LocalDirAllocator$AllocatorPerContext.confChanged(LocalDirAllocator.java:<*>) at org.apache.hadoop.fs.LocalDirAllocator$AllocatorPerContext.getLocalPathToRead(LocalDirAllocator.java:<*>) at org.apache.hadoop.fs.LocalDirAllocator.getLocalPathToRead(LocalDirAllocator.java:<*>) at org.apache.hadoop.mapred.YarnChild.configureLocalDirs(YarnChild.java:<*>) at org.apache.hadoop.mapred.YarnChild.configureTask(YarnChild.java:<*>) at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:<*>)"
5f58c400,"Diagnostics report from attempt_<*>_<*>_m_<*>_<*>: Error: java.lang.NumberFormatException: For input string: ""LoGvv"" at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:<*>) at java.base/java.lang.Integer.parseInt(Integer.java:<*>) at java.base/java.lang.Integer.parseInt(Integer.java:<*>) at org.apache.hadoop.conf.Configuration.getInt(Configuration.java:<*>) at org.apache.hadoop.fs.ChecksumFileSystem.setConf(ChecksumFileSystem.java:<*>) at org.apache.hadoop.util.ReflectionUtils.setConf(ReflectionUtils.java:<*>) at org.apache.hadoop.util.ReflectionUtils.newInstance(ReflectionUtils.java:<*>) at org.apache.hadoop.fs.FileSystem.createFileSystem(FileSystem.java:<*>) at org.apache.hadoop.fs.FileSystem.access$<*>(FileSystem.java:<*>) at org.apache.hadoop.fs.FileSystem$Cache.getInternal(FileSystem.java:<*>) at org.apache.hadoop.fs.FileSystem$Cache.get(FileSystem.java:<*>) at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:<*>) at org.apache.hadoop.fs.FileSystem.getLocal(FileSystem.java:<*>) at org.apache.hadoop.fs.LocalDirAllocator$AllocatorPerContext.confChanged(LocalDirAllocator.java:<*>) at org.apache.hadoop.fs.LocalDirAllocator$AllocatorPerContext.getLocalPathToRead(LocalDirAllocator.java:<*>) at org.apache.hadoop.fs.LocalDirAllocator.getLocalPathToRead(LocalDirAllocator.java:<*>) at org.apache.hadoop.mapred.YarnChild.configureLocalDirs(YarnChild.java:<*>) at org.apache.hadoop.mapred.YarnChild.configureTask(YarnChild.java:<*>) at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:<*>)"
0e4be9d6,<*> failures on node <*>
ae71f36c,Added attempt_<*>_<*>_m_<*>_<*> to list of failed maps
cdaa1ea7,Diagnostics report from attempt_<*>_<*>_m_<*>_<*>:
424f14ec,"attempt_<*>_<*>_m_<*>_<*> transitioned from state FAIL_FINISHING_CONTAINER to FAILED, event type is <*> and <*>"
99b8a8a2,"Assigning container Container: [ContainerId: container_<*>_<*>_<*>_<*>, AllocationRequestId: <*>, Version: <*>, NodeId: <*> NodeHttpAddress: <*> Resource: <memory:<*>, vCores:<*>>, Priority: <*>, Token: Token { kind: ContainerToken, service: <*> }, ExecutionType: GUARANTEED, ] to fast fail map"
a275647a,Assigned from earlierFailedMaps
a0d2d61b,Blacklisted host <*>
20ca077e,Update the blacklist for application_<*>_<*>: blacklistAdditions=<*> blacklistRemovals=<*>
350cb79a,"Ignore blacklisting set to true. Known: <*>, Blacklisted: <*>, <*>%"
db25a59d,Job failed as tasks failed. failedMaps:<*> failedReduces:<*> killedMaps:<*> killedReduces: <*>
6f60dede,KILLING attempt_<*>_<*>_m_<*>_<*>
0ff891cc,Could not deallocate container for task attemptId attempt_<*>_<*>_r_<*>_<*>
f1471d59,Could not delete hdfs://2f08f873c798:<*>/efghterasort0kTDppcE65/_qke7koutput/_temporary/<*>/_temporary/attempt_<*>_<*>_m_<*>_<*>
8b21ced7,Set historyUrl to http://cf6e5ca2712c:<*>/jobhistory/job/job_<*>_<*>
629ed051,Setting job diagnostics to Task failed task_<*>_<*>_m_000000Job failed as tasks failed. failedMaps:<*> failedReduces:<*> killedMaps:<*> killedReduces: <*>
c2c7d257,History url is http://cf6e5ca2712c:<*>/jobhistory/job/job_<*>_<*>
02796010,"Stopped ServerConnector@7b5b5bfe{HTTP/<*>.<*>, (http/<*>.<*>)}{<*>}"
